packages/cli/src/commands/type-check.js

```diff
@@ -55,13 +56,17 @@ export const handler = async ({ sides, verbose, prisma, generate }) => {
   const typeCheck = async () => {
     let conclusiveExitCode = 0
 
+    const yarnVersion = await getCmdMajorVersion('yarn')
+
     const tscForAllSides = sides.map((side) => {
       const projectDir = path.join(getPaths().base, side)
       // -s flag to suppress error output from yarn. For example yarn doc link on non-zero status.
       // Since it'll be printed anyways after the whole execution.
       return {
         cwd: projectDir,
-        command: `yarn -s tsc --noEmit --skipLibCheck`,
+        command: `yarn ${
```

Comment:
@jtoar what if we created a helper for running yarn commands so all the logic is in one place? Something like `runYarnCommand`